samples/llama_index_quick_start.ipynb

```diff
@@ -0,0 +1,1014 @@
+    "        \"type\",\n",
+    "        \"country\",\n",
+    "        \"date_added\",\n",
+    "        \"release_year\",\n",
+    "        \"rating\",\n",
+    "        \"duration\",\n",
+    "        \"listed_in\",\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "fr1rP6KQ-8ag"
+   },
+   "source": [
+    "#### Create a LlamaIndex `Index`\n",
+    "\n",
+    "An `Index` allows us to quickly retrieve relevant context for a user query. They are used to build `QueryEngines` and `ChatEngines` over which a user can get answers to their queries.\n",
+    "For a list of indexes that can be built in LlamaIndex, see [Index Guide](https://docs.llamaindex.ai/en/stable/module_guides/indexing/index_guide/).\n",
+    "\n",
+    "A `VectorStoreIndex`, can be built using the `AlloyDBVectorStore`. You can also use the `AlloyDBDocumentStore` and `AlloyDBIndexStore` to persist documents and index metadata. These modules can be used to build other `Indexes`.\n",
+    "For a detailed python notebook on this, see [LlamaIndex Doc Store Guide](https://github.com/googleapis/llama-index-alloydb-pg-python/blob/main/samples/llama_index_doc_store.ipynb).\n",
+    "\n",
+    "Now, add the documents data into the vector table. Here is a code example to load the first 100 documents in the list."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "CTks8Cy--93B"
+   },
+   "outputs": [],
+   "source": [
+    "from llama_index.core import StorageContext, VectorStoreIndex\n",
+    "\n",
+    "storage_context = StorageContext.from_defaults(vector_store=vector_store)\n",
+    "docs_to_load = documents[:100]\n",
+    "index = VectorStoreIndex.from_documents(\n",
+    "    documents, storage_context=storage_context, show_progress=True\n",
+    ")\n",
+    "\n",
+    "# ! Uncomment the following line to load all 8,800+ documents to the database vector table with calling the embedding service.\n",
+    "# Will incur costs to create embeddings for each record\n",
+    "# docs_to_load = documents[100::]\n",
+    "# vector_store.add_documents(docs_to_load)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Query over indexed data\n",
+    "\n",
+    "A query engine takes in a natural language query and returns a rich response. It is built on one of the indexes, see [Query Engine Guide](https://docs.llamaindex.ai/en/stable/module_guides/deploying/query_engine/).\n",
+    "\n",
+    "You can compose multiple query engines to achieve more advanced querying, see [Query Engine usage patterns](https://docs.llamaindex.ai/en/stable/module_guides/deploying/query_engine/usage_pattern/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query_engine = index.as_query_engine()\n",
+    "op = query_engine.query(\"What was the cast in Blood and Water.\")\n",
```

Comment:
We could use a better question for vector search, since regular SQL queries can find the cast. Maybe somehting like "List shows that are about teenagers"